TADashboard/src/main/java/page_objects/DataProfilesPage.java

```diff
@@ -33,4 +33,9 @@ public boolean isCheckBoxDisplayed(String profileName) {
     public void clickAddNewLink() {
         lnkAddNew.click();
     }
+
+    public boolean areDataProfilesVisible(String dataProfile) {
```

Comment:
you are just checking one data profile at a time, why do you use `are`?

Reply:
I fixed it. Please check again